composer.json

```diff
@@ -13,8 +13,8 @@
   ],
   "require": {
     "cweagans/composer-patches": "^1.6",
-    "drush/drush": "^10.3 || ^11",
-    "google/apiclient": "^2.8"
+    "drush/drush": "^10.3 || ^11 || ^12",
+    "google/apiclient": "^2.15"
```

Comment:
Hope we do not need to update anything in the [GoogleSheetApi.php](https://github.com/acquia-pso/dst-entity-generate/blob/master/src/Services/GoogleSheetApi.php) implementation. Please validate and update.